### PR TITLE
feat: Names charts

### LINKED ISSUE #533: Names charts
### Please describe the problem that your request should resolve.
Display activated (or claimed) names

### Describe the solution you'd like
Chart displaying total number of names on time axis

### Is it already possible to achieve the same outcome in any other way? If so, how?
MDW Tracking https://github.com/aeternity/ae_mdw/issues/1612

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #533

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] add names chart

### CODE REVIEW

**Review by Liubov-crypto — CHANGES_REQUESTED** [review]
In general LGTM. 
But is this correct aligning? 
https://github.com/aeternity/aescan/assets/69896204/5f3f322c-0e4a-4b76-bdec-79966d3488ab

**Comment by janmichek** [comment]
> In general LGTM. But is this correct aligning? https://github.com/aeternity/aescan/assets/69896204/5f3f322c-0e4a-4b76-bdec-79966d3488ab

Yeah, that's what is the data. I can do better with printing the labels for Y axis. There should be only full numbers, no fractions.

**Comment by janmichek** [comment]
At least I fixed it in this fashion
image
Not super satisfied about it, but that's the data we get. We can consult if you guys have some bright ideas .)

**Comment by Liubov-crypto** [comment]
I found that for some dates it will not shown any data, just red dot: 
tr

https://github.com/aeternity/aescan/assets/69896204/83def734-9fa9-4017-8840-2c86160b7ee5

@janmichek please check it

**Comment by janmichek** [comment]
> I found that for some dates it will not shown any data, just red dot: tr
> 2024-01-08.6.00.55.mov
> 
> @janmichek please check it

Good catch, fixed it.

**Review by Liubov-crypto — APPROVED** [review]
Fixed, LGTM.

File: src/stores/names.js [hunk 1]
```diff
@@ -73,6 +74,26 @@ export const useNamesStore = defineStore('names', () => {
     rawRecentlyActivatedNames.value = data.data
   }
 
+  // async function fetchNamesStatistics(slug) {
+  //   namesStatistics.value = null
+  //   const { data } = await axios.get(`${MIDDLEWARE_URL}/v3/statistics/names${slug || '?limit=8&interval_by=day'}`)
+  //   namesStatistics.value = data.data.reverse()
+  //   // namesStatistics.value = data.data.slice(1).reverse()
+  // }
```
Comment (michele-franchi) [inline src/stores/names.js:82]: Is this commented function useful or can it be removed?

File: src/stores/oracleDetails.js [hunk 2]
```diff
@@ -40,11 +40,13 @@ export const useOracleDetailsStore = defineStore('oracleDetails', () => {
   async function fetchOracle() {
     const { data } = await axios.get(`${MIDDLEWARE_URL}/v2/oracles/${oracleId.value}?tx_hash=true`)
     rawOracle.value = data
+    console.log('1 rawOracle.value', rawOracle.value)
```
Comment (michele-franchi) [inline src/stores/oracleDetails.js:43]:
```suggestion
```

File: src/stores/oracleDetails.js [hunk 3]
```diff
@@ -40,11 +40,13 @@ export const useOracleDetailsStore = defineStore('oracleDetails', () => {
...
   }
 
   async function fetchLastExtendedTx() {
     const { data } = await axios.get(`${MIDDLEWARE_URL}/v2/txs?direction=backward&limit=1&type=oracle_extend&oracle=${oracleId.value}`)
     lastExtendedTx.value = data.data?.[0]
+    console.log('3 lastExtendedTx.value', lastExtendedTx.value)
```
Comment (michele-franchi) [inline src/stores/oracleDetails.js:49]:
```suggestion
```

File: src/stores/oracleDetails.js [hunk 4]
```diff
@@ -57,6 +59,7 @@ export const useOracleDetailsStore = defineStore('oracleDetails', () => {
     } catch (e) {
       rawEvents.value = null
     }
+    console.log('2 rawEvents.value', rawEvents.value)
```
Comment (michele-franchi) [inline src/stores/oracleDetails.js:62]:
```suggestion
```

**Review by michele-franchi — APPROVED** [review]
Please remove some dev lines and it will look good!

**Pull request merged**